README.md

```diff
@@ -14,9 +18,103 @@ Communicate information about background jobs.
 
 1. Run `bundle install` from your shell.
 
+1. Generate migrations so `FrenchToast` can render information when javascript isn't available.
+
+```
+rails generate french_toast:install
+rake db:migrate
+```
+
 ## Usage
 
-TODO
+### General French Toast
+
+Simplest thing you can do is use the default french toast helper in a view
```

Comment:
It's spelt FrenchToast in other places. Should this be FrenchToast, or maybe `#french_toast`?